src/index.html

```diff
@@ -1,12 +1,229 @@
 <!doctype html>
-<html lang="en">
+<html lang="ru">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="d">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="head">
+      <a href="#home" class="head__logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li>
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li>
+            <a href="#" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+          <li>
+            <a href="#" class="nav__link is-hover" data-qa="nav-hover">Ноутбуки и компьютеры</a>
+          </li>
+          <li>
+            <a href="#" class="nav__link">Гаджеты</a>
+          </li>
+          <li>
+            <a href="#" class="nav__link">Планшеты</a>
+          </li>
+          <li>
+            <a href="#" class="nav__link">ФОТО</a>
+          </li>
+          <li>
+            <a href="#" class="nav__link nav_last">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
```

Comment:
For readability, add blank lines to separate large or logical code blocks, add them in all places in your code.
```suggestion
    </header>\
    
    <main class="main">
```